content/4_Gold/Divisibility.mdx

```diff
@@ -412,6 +412,75 @@ print("\n".join(str(i) for i in ans))
 </PySection>
 </LanguageSection>
 
+### Sum of divisors
+
+<FocusProblem problem = "sumdiv"/>
+
+The sum of all divisors of the integer $N$ can be efficiently computed in $\mathcal{O}(\sqrt{N})$ using [Divisor function](https://en.wikipedia.org/wiki/Divisor_function). The complexity
+comes from the prime factorization of $N$.
+
+The formula states that: Let $d_1^{p_1}, d_2^{p_2}, ..., d_k^{p_k}$ be the prime factorization of $N$, then the divisors' sum is $S = \prod_{i=1}^{k}\frac{d_i^{p_i+1} - 1}{d_i -1}$.
+
+<LanguageSection>
+<CPPSection>
+```cpp
+#include <bits/stdc++.h>
+
+using namespace std;
+
+ifstream in("sumdiv.in");
+ofstream out("sumdiv.out");
+
+const int MOD = 9901;
+
+int n, m;
+// BeginCodeSnip{Binary Exponentiation}
+int exp(int x, int y) {
+	int aux = x, ans = 1;
+	for (int bit = 0; (1 << bit) <= y; bit++) {
+		if ((1 << bit) & y) { ans = ans * aux % MOD; }
+		aux = aux * aux % MOD;
+	}
+	return ans;
+}
+// EndCodeSnip
+
+// BeginCodeSnip{Modular Inverse}
```

Comment:
would prefer if you used the functions in the modular arithmetic module instead if writing your own

Reply:
Done